alf/environments/suite_dmc.py

```diff
@@ -0,0 +1,174 @@
+
+    def close(self):
+        return self._env.close()
+
+    def render(self, mode='rgb_array'):
+        return self._env.render(mode)
+
+    def seed(self, seed):
+        return self._env.seed(seed)
+
+    def wrapped_env(self):
+        return self._env
+
+
+@alf.configurable
+def dmc_loader(environment_name,
+               domain_name='cheetah',
+               task_name='run',
+               seed=1,
+               from_pixels=True,
+               pre_transform_image_size=100,
+               action_repeat=1,
+               env_id=None,
+               discount=1.0,
+               max_episode_steps=1000,
+               gym_env_wrappers=(),
+               alf_env_wrappers=()):
+    """ Load the MuJoCo environment through dmc2gym
+
+    This loader will not take environment_name, instead please use domain_name and tesk_name.
+    For installation of dmc2gym, see https://github.com/denisyarats/dmc2gym.
+    For installation of DMControl, see https://github.com/deepmind/mujoco.
+    For installation of MuJoCo200, see https://roboti.us.
+    If there are any bug regarding env_info, please include AlfEnvironmentDMC2GYMWrapper
+    in the alf_env_wrappers.
+
+    Args:
+        environment_name (str): Do not use this arg, this arg is here to
+            match up with create_environment.
+        domain_name (str): The name of MuJoCo domain that is used.
+        task_name (str): The name of task we want the agent to do in the
+            current MuJoCo domain.
+        seed (int): Random seed for the environment.
+        from_pixels (boolean): Output image if set to True.
+        pre_transform_inage_size (int): The height and width of the output 
+            image from the environment.
+        action_repeat (int): Action repeat of gym environment.
+        env_id (str): The environment id generated form domain_name, task_name
+            and seed.
+        discount (float): Discount to use for the environment.
+        max_episode_steps (int): The maximum episode step in the environment.
+            Note that the episode length in the alf will be max_episode_steps/action_repeat.
+        gym_env_wrappers (Iterable): Iterable with references to gym_wrappers
+            classes to use directly on the gym environment.
+        alf_env_wrappers (Iterable): Iterable with references to alf_wrappers
+            classes to use on the ALF environment. 
+        image_channel_first (bool): whether transpose image channels to first dimension.
+
+    Returns:
+        A wrapped AlfEnvironment
+    """
+
+    gym_env = dmc2gym.make(
+        domain_name=domain_name,
+        task_name=task_name,
+        seed=seed,
+        visualize_reward=False,
+        from_pixels=from_pixels,
+        episode_length=max_episode_steps,
+        height=pre_transform_image_size,
+        width=pre_transform_image_size,
+        frame_skip=action_repeat)
+    return wrap_env(
+        gym_env,
+        env_id=env_id,
```

Comment:
env_id taken by wrap_env should be int. But according to the comment of this function, it is str.